web/app_dev.php

```diff
@@ -1,28 +1,15 @@
 <?php
```

Comment:
`web/app_end2end.php` and `web/app_continuumtest.php` are missing, if in doubt they should be identical to `web/app_prod.php`

Reply:
Have added